### DISCUSSION TITLE
Set outer window width?

### DISCUSSION THREAD

**Discussion by AyashiNoCeres** [emilk/egui#190] [Q&A] [answered]
I may just be looking in the wrong place but I've searched and searched and have been unable to find how to set the initial size of the window on startup

**Comment by emilk** [comment, marked as answer]
I've seem to have forgotten about it - until now! I added it in https://github.com/emilk/egui/commit/02a65132e42e26b28351a7fcd06d68d322b4e334

**Reply by AyashiNoCeres** [reply]
Many thanks for this! 

And thank you for this great library!
